aio_pika/channel.py

```diff
@@ -210,6 +210,8 @@ def _on_return(self, message: aiormq.types.DeliveredMessage):
 
         if auto_delete and durable is None:
             durable = False
+        
+        type = ExchangeType(type)
```

Comment:
This was done intentionally. The fact is that in RabbitMQ, extensions use custom Exchange types other than this enumeration. So this check breaks the use of third-party Exchange types.

Reply:
Then you have the wrong type hint. I tried to send a string and got an error. Look [#260](https://github.com/mosquito/aio-pika/issues/260).

Reply:
@mosquito ?